### PR TITLE
Issue 100

### PR DESCRIPTION
**Pull request by sebastianbot6969**

### CODE REVIEW

File: report/src/main.tex
Comment (sebastianbot6969) [inline report/src/main.tex:1]: delete onecolumn

File: report/src/sections/aa-appendix/02-monotone.tex
Comment (sebastianbot6969) [inline report/src/sections/aa-appendix/02-monotone.tex:1]: Should everything be outcommented

File: report/src/sections/aa-appendix/03-galois.tex
Comment (sebastianbot6969) [inline report/src/sections/aa-appendix/03-galois.tex:1]: seems something is missing?

File: report/src/sections/aa-appendix/02-monotone.tex
Comment (CasperStaahl) [reply report/src/sections/aa-appendix/02-monotone.tex:1]: yes for now

File: report/src/sections/04-theory/01-abstract-domains.tex [hunk 1]
```diff
@@ -1,58 +1,71 @@
...
 In the following we do not distinguish between regular expressions and their languages.
 
-In regards to abstract interpretation regulars expressions are not without their own quirks, as expressed in the following theorem.
+The lattice of $(REG, \subseteq, \cup, \cap)$ is a lattice but not a complete one as the set of regular languages is transfinite and closed under finite union and intersection, but not closed under transfinite union and intersection.
+This is a problem: Essentially we can not employ Kleene fixed-point theorem (\autoref{thm:kleene_finite} or \autoref{thm:kleene_scott}) to prove termination of our analysis.
+
+\subsubsection{Abstract domain of numbers as linear inequalities}\label{subsubsec:abstract_domains_numbers}
 
-The lattice of regular expressions $(REG, \subseteq, \cup, \cap)$ is not suited for abstract interpretation, as expressed by \autoref{thm:reg-lattice}.
+To represent the abstract domain of the family of SQL number data types when have chosen compound linear inequalities with mixed integer solutions that are algorithmically solvable with MILP solvers (henceforth linear inequalities).
```
Comment (sebastianbot6969) [inline report/src/sections/04-theory/01-abstract-domains.tex:21]: Write out what MILP solvers are

**Review by ironmand123 — APPROVED** [review]

**Pull request merged**